adr/ADR-20.md

```diff
@@ -176,31 +180,31 @@ type ObjectMeta struct {
 }
 ```
 
-### ObjectInfo 
+### ObjectInfo
 
-Object Info is meta plus instance information. 
-The fields in ObjectMeta are serialized in line as if they were 
-direct fields of ObjectInfo 
+Object Info is meta plus instance information.
+The fields in ObjectMeta are serialized in line as if they were
+direct fields of ObjectInfo
 
 ```go
 type ObjectInfo struct {
     ObjectMeta
-    
+
```

Comment:
not sure these spaces are desired, also below and above

Reply:
Those spaces were removed in this PR, not added (by markdown linter). I have no idea why you would want them here, but the docs renders the same.

Reply:
oh god, need more coffee sorry, thought you were adding them lol.

Reply:
Right I see whats happening, you're removing spaces fine - but I think the entire line should be removed.

Reply:
Yeah, you're right, removed them entirely.
It was just linter cleaning things up. I didn't want to change more than required for this PR, as I will cleanup the ADR as follow up :).